dorm/dorm/Presenter/RoomCollectionView/RoomCollectionViewCell.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  RoomCollectionViewCell.swift
+//  dorm
+//
+//  Created by Keum MinSeok on 2022/07/20.
+//
+
+import UIKit
+
+class RoomCollectionViewCell: UICollectionViewCell {
```

Comment:
```suggestion
final class RoomCollectionViewCell: UICollectionViewCell {
```

더이상 상속되지 않는다는 것을 명시하기 위해 final을 추천해요